concent_api/verifier/tasks.py

```diff
@@ -119,12 +127,12 @@ def blender_verification_order(
             unpack_archive(
```

Comment:
@cameel comment from https://github.com/golemfactory/concent/pull/514#discussion_r195780046

> Will it crash if any of the unpacked files already exists? It should. We prefer to crash than to overwrite anything.

Reply:
No, currently it will overwrite.

Reply:
I don't see an option to set it in unpacking function, so I would have to manually extract files list and check if any file already exists. But just few lines before I am doing `clean_directory`, so there should not be any files there. Do you think we need this extra check ?

Reply:
It's just a sanity check. It would be better to have this but we can skip it if it adds complexity or requires a lot of extra work in this particular case.

Reply:
Should it just raise exception or schedule `verification_result` task with error ?

Reply:
Both :)

Ideally you would raise an exception, that exception would be caught just before escaping the task and it would result in the error being reported to Sentry and `verification_result` being sent to the control cluster. One important thing is that if you schedule a `verification_result`, the task must not be retried any more.

Thats a bit like the decorator catching and reporting exceptions but not quite because it's specific to verification (it reports the result) and it does not reraise the exception.

Reply:
:+ done